File: backend/AdPlatform/DTOs/Users/UserDto.cs
namespace AdPlatform.DTOs.Users;

public class UserDto
{
    public int Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Surname { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string Phone { get; set; } = string.Empty;
    public string AvatarSrc { get; set; } = string.Empty;
    public DateTimeOffset RegistrationDate { get; set; }
}

File: frontend/src/types/DTOs/userDto.ts
import { UserLinkDto } from '@/types/DTOs/userLinkDto.ts';

export interface UserDto {
  id: number;
  name: string;
  surname: string;
  email: string;
  phone: string;
  avatarSrc: string;
  registrationDate: string;
  links: UserLinkDto[];
}
